src/include/daos/object.h

```diff
@@ -94,20 +134,6 @@ enum {
 	DAOS_OC_EC_K4P2_L32K,	/* Erasure code, 4 data cells, 2 parity cells,
 				 * cell size 32K.
 				 */
```

Comment:
I think that this patch probably broke those special classes. This would have to be fixed.

Reply:
This has been fixed.